.changes/unreleased/Fixes-20221002-081603.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Fixes
+body: Ignore case when matching relations ([#140](https://github.com/starburstdata/dbt-trino/issues/140)
```

Comment:
Add only `Ignore case when matching relations` to the body.

Reply:
done